mix.exs

```diff
@@ -120,6 +122,18 @@ defmodule Antikythera.Mixfile do
     ]
   end
 
+  defp package() do
+    [
+      licesnses:   ["Apache 2.0"],
+      maintainers: ["antikythera-gr@access-company.com"],
+      links:       %{"GitHub" => @github_url},
+      files:       [
+        "assets", "core", "eal", "lib", "local", "priv", "rel",
```

Comment:
The png file in `"assets"` dir looks unnecessary to me. Can we omit `"assets"` from the hex package?